qiskit/ignis/characterization/coherence/fitters.py

```diff
@@ -16,6 +16,7 @@
 Fitters of characteristic times
 """
 
+import numpy as np
 from .. import BaseCoherenceFitter
```

Comment:
Relative import in parent package.

Reply:
I like this because it allows easily using this package without it being pip'ed